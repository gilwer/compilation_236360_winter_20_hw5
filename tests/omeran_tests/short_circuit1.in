bool f(){
	print("Shalom, shalom - ");
	return false;
}

bool g(){
	print("ve-ein shalom.");
	return true;
}

void main(){
	bool a = f() and (g() or f());
	bool c = g() or (true or (false and f()));
}
