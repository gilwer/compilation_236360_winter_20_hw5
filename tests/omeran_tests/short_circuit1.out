Shalom, shalom - 
ve-ein shalom.
